_lifeofinternatpuppet/post_16.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Week 16
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 16
+---
+
+Hello Everyone!
+
+Not too long to Christmas now, of course?
+
+This has been the busiest week of my placement so far, before the team wraps up for Christmas holidays!
+
+Some of the key highlights included:-
+
+**Module Release Prep**
+- [puppetlabs-motd (IAC-1308)](https://tickets.puppetlabs.com/browse/IAC-1308) (v4.3.0)
```

Comment:
tagmail and tomcat are listed twice, one with pr and one with the release ticket, the duplication is unnecessary